cypress/support/fragments/inventory/inventorySearchAndFilter.js

```diff
@@ -724,6 +732,11 @@ export default {
     cy.do(browseSearchAndFilterInput.choose(option));
   },
 
+  verifySearchOption(option, selectValue) {
```

Comment:
Inacurate method's nameю You not only verify but choose option

Either rename method or divide it
BTW In https://github.com/folio-org/stripes-testing/pull/2165 PR, I've created method
 InventorySearchAndFilter.verifySelectedSearchOption(option);

So in test you just can use it like   

  InventorySearchAndFilter.selectSearchOptions( testData.instanceQuerySearchOption, '');
  InventorySearchAndFilter.verifySelectedSearchOption(testData.instanceQuerySearchValue);

It will be more understandable from test

Reply:
follow your suggestion to divide into methods